src/plugins/cal-events/cal-events.js

```diff
@@ -188,16 +188,6 @@ var componentName = "wb-calevt",
 			if ( !directLinking ) {
 				linkId = event.id || wb.getId();
 				event.id = linkId;
-
-				/*
-				 * Fixes IE tabbing error:
-				 * http://www.earthchronicle.com/ECv1point8/Accessibility01IEAnchoredKeyboardNavigation.aspx
-				 */
-
-				// TODO: Which versions of IE should this fix be limited to?
-				if ( wb.ie ) {
```

Comment:
Removed these kinds of conditions since ``wb.ie`` isn't truthy in IE11 (technically a bug... but as a result IE11 never tried running them).